src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,105 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private Entry<K, V>[] table;
+    private int size = 0;
+    private Entry<K, V> nullKeyEntry;
+
+    @SuppressWarnings("unchecked")
+    public MyHashMap() {
+        table = (Entry<K, V>[]) new Entry[INITIAL_CAPACITY];
+    }
+    
+    private static class Entry<K, V> {
```

Comment:
Remember to parameterize the Node class (here Entry). Class `Entry` should take key and value as parameters, so we should parameterize it to prevent problems with data types.